demos/projects/ESPRESSIF/esp32/config/FreeRTOSConfig.h

```diff
@@ -0,0 +1,142 @@
+/* Copyright (c) Microsoft Corporation.
```

Comment:
Remove this file.

Reply:
The whole file was removed, as per our sync offline.